src/util/redisConnector.js

```diff
@@ -0,0 +1,36 @@
+const redis = require('redis');
+const log = require('../logger');
+
+let a = 1;
+const host = process.env.REDIS_HOST || 'localhost';
+const port = process.env.REDIS_PORT || 6379;
+const password = process.env.REDIS_PASSWORD || '';
+const DBConnector = {
+  redisInstance: redis.createClient({
+    socket: {
+      host,
+      port,
+    },
+    password,
+  }),
+  async getRedisInstance() {
+    if (!DBConnector.redisInstance?.isOpen) {
```

Comment:
What if `redisInstance` is undefined we will still get truthy value here and will try to access undefined value in this block

Reply:
Added a check for redisInstance